HW4_Vaganova/protein_analysis_tool.py

```diff
@@ -128,7 +166,10 @@ def get_mrna(seq: str) -> str:
     :param seq: protein seq in 1-letter encoding (str)
     :return: potential encoding mRNA sequences with multiple choice for some positions (str)
     """
-    pass
+    mrna_seq = str()
+    for res in amino_acid_seq:
```

Comment:
In my opinion is better to use variable seq everywhere in script for same things, like here. Maybe replace?